Library/Formula/metashell.rb

```diff
@@ -1,8 +1,9 @@
 class Metashell < Formula
   desc "Metaprogramming shell for C++ templates"
   homepage "https://github.com/sabel83/metashell"
-  url "https://github.com/sabel83/metashell/archive/v2.0.0.tar.gz"
-  sha256 "5388a6d92b0daecc5e2092554041b70905a1d8ee08bd5ca52c756bde35b4e455"
+  url "https://github.com/sabel83/metashell/archive/v2.1.0.tar.gz"
+  sha256 "64d3680a536a254de8556a9792c5d35e6709f2f347d7187614271123d87246ee"
+  depends_on "cmake" => :build
```

Comment:
This line was actually in the right place before. Dependencies go underneath the `bottle` block.